app/src/main/java/com/humberto/tasky/agenda/data/event/EventMappers.kt

```diff
@@ -34,13 +42,9 @@ fun AgendaItem.Event.toEventEntity(): EventEntity {
         to = to.toInstant().toEpochMilli(),
         remindAt = remindAtEpoch,
         isUserEventCreator = isUserEventCreator,
-        attendees = attendees.map {
-            it.toLocalAttendee(
-                eventId = id,
-                remindAt = remindAtEpoch
-            )
-        },
-        photoKeys = photos.map { it.key }
+        attendees = attendees.map { it.toLocalAttendee() },
+        photoKeys = photos.map { it.key },
+        host = eventCreator?.userId ?: hostId ?: ""
```

Comment:
not sure about this 😅

Reply:
Don’t accept a nullable type 😄 See my previous comment